cluster/gce/container-linux/configure-helper.sh

```diff
@@ -294,6 +294,7 @@ users:
 clusters:
 - name: local
   cluster:
+    server: https://${1}:443
```

Comment:
The `:443` part shouldn't be necessary since that is the default port for https.